### PR TITLE
GNMI-1.5: Attaching/Deattaching LAG to NI using deviation and updating cleanup order

### PR DESCRIPTION
**Pull request by trathod1**
There are 2 changes -
      1. Attaching/Deattaching LAG using `ExplicitInterfaceInDefaultVRF` deviation
      2. Moving LACP config/cleanup after LAG config/cleanup, considering LAG-interface has dependency over LACP. Based on this change, `t.Cleanup` stack would pop LACP context first for deletion and later interface-lag
    
 "This code is a Contribution to the OpenConfig Feature Profiles project ("Work") made under the Google Software Grant and Corporate Contributor License Agreement ("CLA") and governed by the Apache License 2.0. No other rights or licenses in or to any of Nokia's intellectual property are granted for any other purpose. This code is provided on an "as is" basis without any warranties of any kind."

### CODE REVIEW

**Comment by jjing1122** [comment]
@trathod1 please rebase. thanks

**Comment by trathod1** [comment]
@jjing1122 Rebased branch.

**Comment by jjing1122** [comment]
filed internal presubmit bug b/272066511

File: feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go [hunk 1]
```diff
@@ -192,35 +193,43 @@ func (tc *testCase) configureDUT(t *testing.T) {
...
+		lacp := &oc.Lacp_Interface{Name: ygot.String(tc.aggID)}
+		lacp.LacpMode = oc.Lacp_LacpActivityType_ACTIVE
+
+		lacpPath := d.Lacp().Interface(tc.aggID)
+		fptest.LogQuery(t, "LACP", lacpPath.Config(), lacp)
+		gnmi.Replace(t, tc.dut, lacpPath.Config(), lacp)
+		t.Cleanup(func() {
+			gnmi.Delete(t, tc.dut, lacpPath.Config())
+		})
+	}
```
Comment (jjing1122) [inline feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go:232]: as we discussed offline, other vendors might have dependency of LACP-lag-interface on /lacp/ path configuration. please consider a way to have alternate config/delete order.

Comment (trathod1) [reply feature/gnmi/ate_tests/telemetry_port_speed_test/telemetry_port_speed_test.go]: @jjing1122 I pushed the commit, where I am restoring the original config order. Also, introducing `gnmi.BatchDelete` for both LACP and lag-interface config based on `deviations.AggregateAtomicUpdate`, considering this same deviation was used to configure both LACP and lag-interface together.

**Review by jjing1122 — APPROVED** [review]

**Pull request merged**